app/mailers/admin_notice_mailer.rb

```diff
@@ -0,0 +1,8 @@
+class AdminNoticeMailer < ActionMailer::Base
+  def notice_for_councillor_contribution
+    @admins =  User.where(admin: true)
```

Comment:
Rather than going out to all the admins here, I think we can just send this to the central site owners email address, like we do with abuse reports https://github.com/openaustralia/planningalerts/blob/master/app/mailers/report_notifier.rb#L6